### PR TITLE
WIP oauthclient: add cascading action of token

### PR DESCRIPTION
**Pull request by hachreak**
* Adds cascading action on relationship from token to client.
  (addresses #3218)

Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

**Comment by hachreak** [comment]
ping @jirikuncar

File: invenio/modules/oauthclient/utils.py [hunk 1]
```diff
@@ -125,6 +125,6 @@ def oauth_link_external_id(user, external_id=None):
 
 def oauth_unlink_external_id(external_id):
     """Unlink a user from an external id."""
-    UserEXT.query.filter_by(id=external_id['id'],
+    UserEXT.query.filter_by(id=external_id['id'].encode('utf-8'),
```
Comment (jirikuncar) [inline invenio/modules/oauthclient/utils.py]: Why do you have to encode it? Can we find any better way?

**Comment by hachreak** [comment]
It depends on https://github.com/inveniosoftware/invenio/pull/3314

**Comment by jirikuncar** [comment]
@hachreak has it been integrated in `invenio-oauthclient` completely? If yes, please close this PR. Thanks

**Comment by jirikuncar** [comment]
Solved in https://github.com/inveniosoftware/invenio-oauthclient/blob/master/invenio_oauthclient/models.py#L148

**Pull request closed without merging**